x-pack/test/functional/services/cases/common.ts

```diff
@@ -29,125 +28,11 @@ export function CasesCommonServiceProvider({ getService, getPageObject }: FtrPro
       });
     },
 
-    /**
```

Comment:
The common service began to grow a lot. I separate the functionality to other services.